dev/src/codewind/project/ProjectType.ts

```diff
@@ -139,10 +139,8 @@ export class ProjectType {
     public get canInjectMetrics(): boolean {
         // This should be the job of the capabilities API
         return [
-            ProjectType.InternalTypes.MICROPROFILE,
-            ProjectType.InternalTypes.NODE,
-            ProjectType.InternalTypes.SPRING
-        ].includes(this.internalType);
+            'java', 'nodejs',
```

Comment:
for reviewers: this PR also gives more project types the `injectMetrics` tickbox. The additional projects that will have the tickbox are java projects that are not microprofile or spring (e.g. Open Liberty, which is type `docker`)

Reply:
@mattcolegate when we duplicate this PR into `0.7.0`, we should leave this change out, because this is a feature for `0.8.0` (allow injection into java projects that are not microprofile or spring). (And also we haven't yet made the PFE changes to support this yet)